security-group.tf

```diff
@@ -1,25 +1,34 @@
 resource "aws_security_group" "bastion_ssh" {
-  name_prefix = "${var.bastion_name}-ssh-"
+  name_prefix = "${var.bastion_name}-ssh-egress-"
```

Comment:
I'm a little confused by this naming.  If it's traffic into the bastion (description says inbound), then wouldn't it be `ingress` ?

Reply:
I named the SG to describe both kinds of rules (inbound SSH + all egress). Perhaps I should go the other way and generalize the name to not describe the SG use?

Reply:
I did generalize the SG name, since its description explains inbound SSH and egress.